packages/amplify-e2e-core/src/utils/rds.ts

```diff
@@ -422,7 +520,7 @@ export const deleteDBCluster = async (identifier: string, region: string): Promi
     await client.send(command);
   } catch (error) {
     console.log(error);
-    throw new Error(`Error in deleting RDS cluster ${identifier}: ${error.response.json}`);
+    throw new Error(`Error in deleting RDS cluster ${identifier}: ${error.json}`);
```

Comment:
How do we know the incoming error has a `json` property? If it doesn't will we lose meaningful troubleshooting context?